lib/mixins/property-effects.html

```diff
@@ -82,20 +84,23 @@
    *
    * @param {Object} model Prototype or instance
    * @param {string} type Property effect type
+   * @param {boolean=} cloneArrays Clone any arrays mapped in the map
```

Comment:
Instead of adding a boolean parameter, I would rather see an `Object{cloneArrays:boolean}` per #4864